### ISSUE TITLE
Table Structure fine-tuning Dataset

### ISSUE THREAD

**Issue by emigomez** [PaddlePaddle/PaddleOCR#8663]
- System Environment： Windows with GPU, conda env (paddle_env) installed
- Version：Paddle： master branch  PaddleOCR： pip install paddlepaddle-gpu

I want to do table structure fine tunning with slanet, but I don't have any dataset and I can't obtain any of your 3 recommendations.
The commands that I want to run are:
```
python tools/train.py -c configs/table/SLANet_ch.yml -o Global.checkpoints=./ppstructure/train/ch_ppstructure_mobile_v2.0_SLANet_train/best_accuracy.pdparams

python tools/train.py -c configs/table/SLANet.yml -o Global.checkpoints=./ppstructure/train/en_ppstructure_mobile_v2.0_SLANet_train/best_accuracy.pdparams
```

Can anyone share a small dataset for finetuning with tables, labels and the train_data/table/train.txt, tu run a small inference? in ch or en

**Comment by TasneemVKhan** [comment]
Can anybody please answer this? Need Help
